test/iron-location.html

```diff
@@ -61,15 +61,13 @@
     });
   }
 
-  suite('<iron-location>', function () {
-    var initialUrl;
-    setup(function() {
-      initialUrl = window.location.href;
-    });
-    teardown(function(){
-      window.history.replaceState({}, '', initialUrl);
-    });
+  // A window.history.replaceState wrapper that's smart about baseURI.
+  function replaceState(url) {
+    window.history.replaceState(
+        {}, '', window.location.protocol + '//' + window.location.host + url);
+  }
 
+  function allTheTests() {
```

Comment:
🚲 🏠 It would be nice for readability to know what `allTheTests` entails. Would you consider giving this a name that describes the contents of the function?